src/index.html

```diff
@@ -9,6 +9,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="container">
+      <div class="containerLogo">
+        <a href="#home" class="linkLogo">
+          <img class="logo" src="../src/images/logo.png" 
+            alt="Store logo MOYO">
```

Comment:
Pay attention to the code style
```suggestion
          <img 
            class="logo" 
            src="../src/images/logo.png" 
            alt="Store logo MOYO"
          >
```